hedera-mirror-rest/model/fractionalFee.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Fee from './fee';
+
+class FractionalFee extends Fee {
+  /**
+   * Parses fractional_fee from element in custom_fee.fractional_fees jsonb column
+   */
+  constructor(fractionalFee) {
+    super(fractionalFee);
+    this.amount = fractionalFee.amount;
+    this.amountDenominator = fractionalFee.amount_denominator;
+    this.maximumAmount = fractionalFee.maximum_amount;
+    this.minimumAmount = fractionalFee.minimum_amount;
+    this.netOfTransfers = fractionalFee.net_of_transfers;
+  }
+
+  static AMOUNT = `amount`;
+  static AMOUNT_DENOMINATOR = `amount_denominator`;
```

Comment:
make sure it's consistent when you change the property names in importer domain classes

```suggestion
  static DENOMINATOR = `denominator`;
```

Reply:
This is now fixed.